### DISCUSSION TITLE
Config resource not supported for SEB installation on Windows 7 ultimate

### DISCUSSION THREAD

**Discussion by peaches175** [SafeExamBrowser/seb-win-refactoring#446] [Q&A] [answered]
Hello,

I am an exams administrator dealing with a student who has Windows 7 Ultimate installed on a 64 bit machine.
They installed SEB version 3.0.1 but an error message pops up saying that a configuration resource is not supported.
The resource is prefixed with a filepath starting with seb://... 

Can someone give pointers on what seems to be the issue? And tips on how to resolve this?

Thank you

**Comment by dbuechel** [comment, marked as answer]
Windows 7 isn't officially supported anymore, neither so SEB 3.0.1, thus your student should upgrade both if or as soon as possible. Regarding the error itself: All the specific details can be found in the application logs, see `%LocalAppData%\SafeExamBrowser\Logs`. Please feel free to post them here so we may localize the underlying problem.

**Reply by peaches175** [reply]
Thank you for settling the matter! I shall post the logs if the student persists with this device and OS (hopefully not the case).
